pkg/image/controller/signature/container_image_downloader.go

```diff
@@ -30,9 +32,19 @@ func (s *containerImageSignatureDownloader) DownloadImageSignatures(image *image
 	if err != nil {
 		return nil, err
 	}
-	source, err := reference.NewImageSource(nil, nil)
+	// TODO: We just need to trick ping() to pass and not error, we don't care
+	// whether the registry is secure or not at this point as GetSignatures() use
+	// different client for signature retrieval?
+	sctx := types.SystemContext{
+		DockerInsecureSkipTLSVerify: true,
```

Comment:
`GetSignatures` _when using the lookaside_ uses a client with default settings, which ignores this boolean. It could be argued that that is a bug.

Do note that using the `X-Registry-Supports-Signatures` _is_ affected by this boolean; as soon as this code starts passing real credentials, it will have to also set `DockerInsecureSkipTLSVerify` appropriately instead of hard-conding to `true`.